content/en/docs/instrumentation/python/getting-started.md

```diff
@@ -1,249 +1,265 @@
+    "end_time": "2022-04-28T00:07:55.892331Z",
+    "status": {
+        "status_code": "UNSET"
+    },
+    "attributes": {
+        "roll.value": 4
+    },
+    "events": [],
+    "links": [],
+    "resource": {
+        "telemetry.sdk.language": "python",
+        "telemetry.sdk.name": "opentelemetry",
+        "telemetry.sdk.version": "1.11.1",
+        "telemetry.auto.version": "0.30b1",
+        "service.name": "unknown_service"
+    }
+}
+{
+    "name": "/roll",
+    "context": {
+        "trace_id": "0x48da59d77e13beadd1a961dc8fcaa74e",
+        "span_id": "0x84f8c5d92970d94f",
+        "trace_state": "[]"
+    },
+    "kind": "SpanKind.SERVER",
+    "parent_id": null,
+    "start_time": "2022-04-28T00:07:55.891500Z",
+    "end_time": "2022-04-28T00:07:55.892552Z",
+    "status": {
+        "status_code": "UNSET"
+    },
+    "attributes": {
+        "http.method": "GET",
+        "http.server_name": "127.0.0.1",
+        "http.scheme": "http",
+        "net.host.port": 5000,
+        "http.host": "localhost:5000",
+        "http.target": "/roll?sides=10&rolls=2",
+        "net.peer.ip": "127.0.0.1",
+        "http.user_agent": "curl/7.68.0",
+        "net.peer.port": 53824,
+        "http.flavor": "1.1",
+        "http.route": "/roll",
+        "http.status_code": 200
+    },
+    "events": [],
+    "links": [],
+    "resource": {
+        "telemetry.sdk.language": "python",
+        "telemetry.sdk.name": "opentelemetry",
+        "telemetry.sdk.version": "1.11.1",
+        "telemetry.auto.version": "0.30b1",
+        "service.name": "unknown_service"
+    }
+}
 ```
 
-## Use the OpenTelemetry Collector for traces
+The `parent_id` of `do_roll` is the same is the `span_id` for `/rolldice`,
+indicating a parent-child reletionship!
 
-Although it’s possible to directly export your telemetry data to specific
-backends, you might have more complex use cases such as the following:
+## Send traces to an OpenTelemetry Collector
 
+The [OpenTelemetry Collector](/docs/collector/getting-started/) is a critical
+component of most production code. Some examples of when it's beneficial to use
+a collector:
 
 * A single telemetry sink shared by multiple services, to reduce overhead of
-  switching exporters.
-
+  switching exporters
+* Aggregating traces across multiple services, running on multiple hosts
+* A central place to process traces prior to exporting them to a backend
 
-* Aggregating traces across multiple services, running on multiple hosts.
+Unless you have just a single service or are experimenting, you'll want to use a
+collector in production code.
 
-To enable a broad range of aggregation strategies, OpenTelemetry provides the
-[opentelemetry-collector](https://github.com/open-telemetry/opentelemetry-collector).
-The Collector is a flexible application that can consume trace data and export
-to multiple other backends, including to another instance of the Collector.
+### Configure and run a local collector
```

Comment:
I see this replicated between your .NET PR and the python PR here, would be good to externalise (not sure if this should be part of the "Getting Started" with the collector or if this could be an excerpt or something else that could be reused), with that one could use it for all the other languages as well.

Reply:
Yeah, I was thinking about using subpages here. @chalin any thoughts? I'm fine with duplicating for now and then consolidating later, but whatever is easiest

Reply:
I was just working on https://github.com/open-telemetry/opentelemetry.io/pull/1332 and I am wondering if this section you wrote (and especially that "output to console" collector config) should live somewhere on that getting started page for the collector (could be after those PRs are merged)

Reply:
Yeah, it does make for a good "Get started" experience for just local development before you stand anything up for real. And it's easy as heck to do! Tentatively I'd say including it in #1332 is better than not including it